app/models/donation_chart.rb

```diff
@@ -13,20 +13,25 @@ def self.donation_issues
   end
 
   def self.wtf
-    Issue.all.map do |issue|
-      issue.candidate_issues.map { |candidate_issue| candidate_issue.donations.
-        map { |donation| donation.amount }}.reduce(:+).reduce(:+)
-    end.map do |num|
-      if num.nil?
-        0
-      else
-        num
+    omg = Issue.all.map do |issue|
+        issue.candidate_issues.map { |candidate_issue| candidate_issue.donations.
```

Comment:
Line is too long. [81/80]<br>Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.